crates/services/txpool_v2/src/service/verifications.rs

```diff
@@ -0,0 +1,303 @@
+use crate::{
+    error::Error,
+    ports::{
+        ConsensusParametersProvider,
+        GasPriceProvider,
+        TxPoolPersistentStorage,
+        WasmChecker,
+    },
+    service::{
+        memory::MemoryPool,
+        Shared,
```

Comment:
For example here I would find it more readable if we instead of importing `Shared` redefine the alias in this module.

Reply:
will let @xgreenx  give his advice on this as it's a type he added

Reply:
Up to @AurelienFT )

Reply:
Noted on my note will think about it in the next days to not block the merge.